src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Model/Column.cs

```diff
@@ -434,6 +434,36 @@ public override void Set(SqlDataRecord record, int ordinal, byte? value)
         }
     }
 
+    /// <summary>
+    /// Represents a rowversion type column.
+    /// </summary>
+    /// <remarks>
+    /// The timestamp data type is a synonym for the rowversion data type, and its syntax is now deprecated in SQL.
```

Comment:
Can you explain a little bit about what you meant that it's now deprecated in SQL? We should not be using anything that's deprecated.

We don't have to use rowversion. We could alternatively just generate automatically incremented count or something.

Reply:
Sure. The first "note" in the [rowversion documentation](https://docs.microsoft.com/en-us/sql/t-sql/data-types/rowversion-transact-sql?view=sql-server-ver15) indicates that the timestamp syntax is deprecated.

I used the rowversion type in 1.sql, but I ran into issues when this was translated into C# objects because the [SqlDbType enum](https://docs.microsoft.com/en-us/dotnet/api/system.data.sqldbtype?view=netframework-4.8) does not support RowVersion (only Timestamp).

I added some code in SqlVisitor.cs that translates `RowVersion` to `Timestamp`, which causes the name of the generated column object to be `TimestampColumn` (instead of `RowVersionColumn`). 

I updated the documentation to make this a bit more clear in b7efe7d - but if you have suggestions on how to improve this, I'm all ears.

Reply:
Similar to above, I'm going to merge this PR to my feature branch for now, but this can be revisited after the holidays if needed.